kgbrules/status_rules.py

```diff
@@ -0,0 +1,89 @@
+import re
+
+from .config import MAX_STATUS_LENGTH
+from .config import TYPES
+
+status_pattern = re.compile(
+    """^([^()\s]*)(\s*)\(([^()\s]*)\)(\s*):(\s*)(.*\w)(\s*)$""")
+
+spaces = re.compile("\w*\s+\w*\(|\w+\(.*[^\w].*\):")
+
+_type_string = ", ".join(sorted(TYPES))
+
+
+def _is_merge(status_line):
+    return status_line.startswith("Merge pull request #")
+
+
+def check_status_length(status_line):
+    """Check status length.
+
+    >>> check_status_length("this is short enough")
+
+    >>> check_status_length("TOO_LONG" * 10)
+    'Limit status to 50 characters'
+    """
+    if not _is_merge(status_line):
+        if len(status_line) > MAX_STATUS_LENGTH:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1066)